code/controllers/subsystem/minimap.dm

```diff
@@ -689,6 +689,11 @@ SUBSYSTEM_DEF(minimaps)
 	if(faction == FACTION_NEUTRAL && isobserver(user))
 		faction = allowed_flags == MINIMAP_FLAG_XENO ? XENO_HIVE_NORMAL : FACTION_MARINE
 
+	if(is_xeno && xeno.hive.see_humans_on_tacmap)
+		allowed_flags = MINIMAP_FLAG_ALL
```

Comment:
```suggestion
		allowed_flags |= MINIMAP_FLAG_USCM|MINIMAP_FLAG_PMC|MINIMAP_FLAG_UPP|MINIMAP_FLAG_CLF
```

Alternatively create:
`#define MINIMAP_FLAG_HUMANS MINIMAP_FLAG_USCM|MINIMAP_FLAG_PMC|MINIMAP_FLAG_UPP|MINIMAP_FLAG_CLF` in `code\__DEFINES\minimap.dm` and use `allowed_flags |= MINIMAP_FLAG_HUMANS`